src/test/features/claim/routes/check-and-send.ts

```diff
@@ -87,270 +87,97 @@ describe('Claim issue: check and send page', () => {
-            .expect(res => expect(res).to.be.successful.withText('I believe that the facts stated in this claim are true.'))
-            .expect(res => expect(res).to.be.successful.withText('<input id="signedtrue" type="checkbox" name="signed" value="true"'))
+      context('Validate details for claimant on check-and-send page', () => {
+        [
+          [individualDetails],
+          [soleTraderDetails],
+          [companyDetails],
+          [organisationDetails]
+        ].forEach(([partyDetails]) => {
+          it(`Should validate that a claim made by individual against ${partyDetails.type}`, async () => {
+            draftStoreServiceMock.resolveFind('claim',
+              { claimant: { ...draftStoreServiceMock.sampleClaimDraftObj.claimant, partyDetails: individualDetails },
+                defendant: { ...draftStoreServiceMock.sampleClaimDraftObj.defendant, partyDetails: partyDetails } })
+            feesServiceMock.resolveCalculateIssueFee()
+
+            await request(app)
+              .get(ClaimPaths.checkAndSendPage.uri)
+              .set('Cookie', `${cookieName}=ABC`)
+              .expect(res => expect(res).to.be.successful.withText('<a href="/claim/claimant-individual-details" class="bold">Change <span class="visuallyhidden">full name</span></a>'))
+              .expect(res => expect(res).to.be.successful.withText('Full name'))
+              .expect(res => expect(res).to.be.successful.withText('John Smith'))
+              .expect(res => expect(res).to.be.successful.withText('Statement of truth'))
+              .expect(res => expect(res).to.be.successful.withText('I believe that the facts stated in this claim are true.'))
+              .expect(res => expect(res).to.be.successful.withText('<input id="signedtrue" type="checkbox" name="signed" value="true"'))
+          })
+
+          it(`Should validate that a claim made by soleTrader against ${partyDetails.type}`, async () => {
+            draftStoreServiceMock.resolveFind('claim',
+              { claimant: { ...draftStoreServiceMock.sampleClaimDraftObj.claimant, partyDetails: soleTraderDetails },
+                defendant: { ...draftStoreServiceMock.sampleClaimDraftObj.defendant, partyDetails: partyDetails }
+              })
+            feesServiceMock.resolveCalculateIssueFee()
+
+            await request(app)
+              .get(ClaimPaths.checkAndSendPage.uri)
+              .set('Cookie', `${cookieName}=ABC`)
+              .expect(res => expect(res).to.be.successful.withText('<a href="/claim/claimant-sole-trader-details" class="bold">Change <span class="visuallyhidden">full name</span></a>'))
+              .expect(res => expect(res).to.be.successful.withText('Business name'))
+              .expect(res => expect(res).to.be.successful.withText('Trading as SoleTrader Ltd.'))
+              .expect(res => expect(res).to.be.successful.withText('Statement of truth'))
```

Comment:
I am ok with all mostly based on what we have done in other code but I am not sure how useful it is to test for example same 'Statement of truth' text and same 'I believe that the facts stated in this claim are true' text for each party (4 times). It's good coverage i guess but there will be lot to maintain later. @shrivsksavit I understand you have done based on existing examples but let's get other opinions and see what they think. @robertparkinson @simad

Reply:
I agree with Kiran, checking for the same content 4 times doesn't seem appropriate. The code that displays the claimant and the defendant are not linked. A test that checks that the correct content shows up when each party type is either a claimant or defendant would be sufficient coverage.
The tests you previously were doing that. they were checking that both combinations of content were appearing which seems a more valid test in my mind.

Reply:
accepted and updated please have a look